0000-template.md

```diff
@@ -7,6 +7,9 @@
 > Write one sentence which is a brief description of the feature from a user
 > perspective ("impact on users").
 
+## Customer pains
+Describe in short the customer pains your'e trying to solve.
```

Comment:
This is basically what lines 7 and 8 are about. Maybe we can just add a note about "customer pain" there instead of adding an additional section.

Reply:
IMHO we should separate between the feature description and the customer pains description. 
It helps focus the discussion and creates a clear separation between the WHY and the WHAT.
WHY is the customer pains.
WHAT is how we're planning to solve it.
For someone who sees the RFC for the first time, I believe it will help read the RFC and understand its scope faster